WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/IconListItem.swift

```diff
@@ -0,0 +1,67 @@
+import Kingfisher
+import SwiftUI
+
+/// Represents an icon to be displayed. Base64 or Remote (from web)
+enum Icon {
+    /// Base64 icon
+    case base64(UIImage)
+
+    /// Icon that comes from an URL
+    case remote(URL)
+
+    /// Return an Image for base64 or a FKImage in case of a remote one
+    @ViewBuilder
+    func getImage() -> some View {
+        switch self {
+        case .base64(let image):
+            Image(uiImage: image)
+                .resizable()
+        case .remote(let url):
+            KFImage(url)
+                .resizable()
+        }
+    }
+}
+
+/// Represent a row with a title, subtitle and icon. Used to fill up reports such as What's New in WooCommerce
+///
+struct IconListItem: View {
+    let title: String
+    let subtitle: String
+    let icon: Icon?
+
+    var body: some View {
+        HStack(alignment: .center, spacing: 16) {
+            icon?.getImage()
+                .frame(width: Constants.iconSize.width, height: Constants.iconSize.height)
+            VStack(alignment: .leading, spacing: 2) {
+                Text(title)
+                    .headlineStyle()
+                Text(subtitle)
+                    .foregroundColor(Color(UIColor.gray(.shade30)))
+                    .bodyStyle()
```

Comment:
Can't this be `.secondaryBodyStyle()`?

Reply:
Totally! I've changed it to secondaryBodyStyle right now 👍 
Done! ✅